doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -4,6 +4,87 @@
  Proof handling
 -------------------
```

Comment:
I don't think that works.  Almost everything in the chapter can be used non-interactively.  How about just "Proofs"?

Reply:
From https://coq.github.io/doc/master/refman/language/core/definitions.html#assertions-and-proofs:

>An assertion states a proposition (or a type) of which the proof (or an inhabitant of the type) is interactively built using tactics. The interactive proof mode is described in Chapter Proof handling

If you have a problem with "interactive" because it also works in batch mode (despite it still being an interactive process between a proof engine and tactics), we can remove the word "interactive" (although it is consistently used elsewhere) but it's important to keep "proof mode" IMHO.

Reply:
I'll make it "Proof mode" for now.  "interactive" seems appropriate for distinguishing between coqtop and coqc, for example, and perhaps in a few places to emphasize that Coq is primarily a proof assistant, not an automatic prover.

I think "proof editing mode" is also too wordy, it would be better just to say "proof mode".  It seems a stretch to consider it "editing" just because there are `Back`/`Undo` commands.

What do you think?  If agreeable, I will update uses of "interactive" and "proof editing mode".

Reply:
Yes, I agree with your proposed change.